pom.xml

```diff
@@ -2717,6 +2717,8 @@
                                     -Xep:NullOptional:ERROR \
                                     -Xep:NullableOptional:ERROR \
                                     -Xep:NullablePrimitive:ERROR \
+                                    <!-- requires https://github.com/google/error-prone/issues/4390 before switching to ERROR -->
+                                    -Xep:ObjectEqualsForPrimitives:WARN \
```

Comment:
https://github.com/google/error-prone/issues/4390.
i think i didn't convert any double/float related stuff and updating the code once i spotted this problem.